base_control/Encoder.ino

```diff
@@ -0,0 +1,15 @@
+void handleEncoder(Motor &motor){
```

Comment:
I feel like this should be wrapped on a namespace, and split in .h and .cpp. Also, where do you define moveAll?